examples/dts/sub-orchestrations-with-fan-out-fan-in/README.md

```diff
@@ -0,0 +1,102 @@
+# Portable SDK Sample for Sub Orchestrations and Fan-out / Fan-in
+
+This sample demonstrates how to use the Durable Task SDK, also known as the Portable SDK, with the Durable Task Scheduler to create orchestrations. These orchestrations not only spin off child orchestrations but also perform parallel processing by leveraging the fan-out/fan-in application pattern.
+
+The scenario showcases an order processing system where orders are processed in batches. 
+
+> Note, for simplicity, this code is contained within a single source file. In real practice, you would have 
+
+# Prerequisites
+ - [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli)
+ - [`az durabletask` CLI extension](https://learn.microsoft.com/en-us/cli/azure/durabletask?view=azure-cli-latest)
+
+## Running the Examples
+There are two separate ways to run an example:
+
+- Using the Emulator
+- Using a deployed Scheduler and Taskhub
+
+### Running with a Deployed Scheduler and Taskhub rResource
+1. To create a taskhub, follow these steps using the Azure CLI commands:
+
+Create a Scheduler:
+```bash
+az durabletask scheduler create --resource-group --name --location --ip-allowlist "[0.0.0.0/0]" --sku-capacity 1 --sku-name "Dedicated" --tags "{'myattribute':'myvalue'}"
```

Comment:
I suggest we guide readers to use the emulator rather than creating resources in Azure. That will save them a ton of time and effort when they're still learning. Let's have separate instructions and/or more substantial samples for configuring apps with DTS in Azure.